src/cdp-proxy/reactNativeCDPProxy.ts

```diff
@@ -134,7 +135,10 @@ export class ReactNativeCDPProxy {
     }
 
     private async onApplicationTargetClosed() {
```

Comment:
Do we need this function here if it is not implemented?

Reply:
This function seems to be redundant, I'll remove it.